fetch-npm-node.js

```diff
@@ -1,16 +1,21 @@
 "use strict";
 
-var realFetch = require('node-fetch');
-module.exports = function(url, options) {
-	if (/^\/\//.test(url)) {
-		url = 'https:' + url;
-	}
-	return realFetch.call(this, url, options);
-};
-
-if (!global.fetch) {
-	global.fetch = module.exports;
-	global.Response = realFetch.Response;
-	global.Headers = realFetch.Headers;
-	global.Request = realFetch.Request;
+// add support for jsdomify for running tests
+// <https://github.com/niftylettuce/frisbee>
+// <https://github.com/podio/jsdomify>
+// <https://github.com/github/fetch/issues/238>
+if (typeof window === 'object') {
```

Comment:
Hmm still doesn't add up. If you're running tests in a browser environment should still hit the browser file